r10k.yaml.example

```diff
@@ -1,5 +1,79 @@
-:cachedir: '/var/cache/r10k'
-:sources:
-  :plops:
-    remote: 'git@github.com:my-org/org-shared-modules'
-    basedir: '/etc/puppet/environments'
+---
```

Comment:
I think a first line that says something like, "This is r10k.yaml. This is used to configure the behavior of r10k when run with the "deploy" action.